### PR TITLE
Xena multinode

### PR DESCRIPTION
**Pull request by grzegorzkoper**

### CODE REVIEW

File: etc/kayobe/environments/ci-multinode/kolla/globals.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+# Most development environments will use nested virtualisation, and we can't
+# guarantee that nested KVM support is available. Use QEMU as a lowest common
+# denominator.
+nova_compute_virt_type: qemu
+
+# Reduce the control plane's memory footprint by limiting the number of worker
+# processes to one per-service.
+openstack_service_workers: "1"
+openstack_service_rpc_workers: "1"
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/kolla/globals.yml]: hmm, we should probably drop these when the compute node is bare metal.

File: etc/kayobe/environments/ci-multinode/stackhpc-ci.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+stackhpc_repo_centos_stream_storage_ceph_pacific_version: "{{ stackhpc_pulp_repo_centos_stream_8_storage_ceph_pacific_version }}"
+stackhpc_repo_elasticsearch_logstash_kibana_7_x_version: "{{ stackhpc_pulp_repo_elasticsearch_logstash_kibana_7_x_version }}"
+stackhpc_repo_grafana_version: "{{ stackhpc_pulp_repo_grafana_version }}"
+stackhpc_repo_rabbitmq_erlang_version: "{{ stackhpc_pulp_repo_rabbitmq_erlang_version }}"
+stackhpc_repo_rabbitmq_server_version: "{{ stackhpc_pulp_repo_rabbitmq_server_version }}"
+stackhpc_repo_treasuredata_4_version: "{{ stackhpc_pulp_repo_treasuredata_4_version }}"
+stackhpc_repo_ubuntu_cloud_archive_version: "{{ stackhpc_pulp_repo_ubuntu_cloud_archive_version }}"
+stackhpc_repo_ubuntu_focal_version: "{{ stackhpc_pulp_repo_ubuntu_focal_version }}"
+stackhpc_repo_ubuntu_focal_security_version: "{{ stackhpc_pulp_repo_ubuntu_focal_security_version }}"
+stackhpc_repo_docker_ce_ubuntu_version: "{{ stackhpc_pulp_repo_docker_ce_ubuntu_version }}"
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/stackhpc-ci.yml]: This should be updated to match the same file in the ci-builder environment

**Review by markgoddard — CHANGES_REQUESTED** [review]

**Comment by markgoddard** [comment]
You'll need to cherry-pick this one across: https://github.com/stackhpc/stackhpc-kayobe-config/pull/183

File: etc/kayobe/storage.yml [hunk 3]
```diff
@@ -70,7 +70,8 @@
 
 # List of storage volume groups. See mrlesmithjr.manage-lvm role for
 # format.
-#storage_lvm_groups:
+storage_lvm_groups:
+  - "{{ stackhpc_lvm_group_rootvg }}"
```
Comment (markgoddard) [inline etc/kayobe/storage.yml]: This should be in the environment directory, not the base config

File: etc/kayobe/environments/ci-multinode/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# Multinode Test Environment
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/README.md:1]: Jack has since made some changes to these docs.

Comment (grzegorzkoper) [reply etc/kayobe/environments/ci-multinode/README.md:1]: Should we just merge the multinode-steps branch after we're done ? There will be changes there still. Naming convention will change in terraform.

File: etc/kayobe/environments/ci-multinode/storage.yml [hunk 5]
```diff
@@ -0,0 +1,154 @@
+---
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/storage.yml:1]: Please remove the comments, similar to compute.yml and controllers.yml

File: etc/kayobe/environments/ci-multinode/README.md (hunk 4 above)
Comment (markgoddard) [reply etc/kayobe/environments/ci-multinode/README.md:1]: Yes, that branch should be merged.

**Review by markgoddard — DISMISSED** [review]
Nice work @jackhodgkiss @grzegorzkoper

**Pull request merged**